### PR TITLE
Update styling, add envAtlas projection

### PR DESCRIPTION
**Pull request by slimbuck**
This PR:
- updates texture-tool to use the latest dark style recently exposed by PCUI
- adds the option of envAtlas projection

Screenshot 2022-10-13 at 12 59 38

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
       "requireConfigFile": false
     },
     "rules": {
-      "no-undef": "error"
+      "no-undef": "error",
```
Comment (willeastcott) [inline package.json:31]: This is already enabled in the PlayCanvas config: https://github.com/playcanvas/playcanvas-eslint-config/blob/main/index.js#L58

File: rollup.config.mjs [hunk 2]
```diff
@@ -1,51 +1,43 @@
...
+        'pcui': [PCUI_PATH]
+    }
+};
 
 const externs = [
     'static/playcanvas-logo.png',
     'static/lib',
     'static/textures',
-    'src/styles.css',
+    // 'src/styles.css',
```
Comment (willeastcott) [inline rollup.config.mjs:37]: Maybe just delete?

File: src/index.js [hunk 3]
```diff
@@ -1,13 +1,14 @@
 import './main.js';
+import './style.scss';
 
 import { version as textureToolVersion } from '../package.json';
-import { version as pcuiVersion } from '@playcanvas/pcui/package.json';
-import { version as engineVersion } from 'playcanvas/package.json';
+// import { version as pcuiVersion } from 'pcui';
+// import { version as engineVersion } from 'playcanvas';
```
Comment (willeastcott) [inline src/index.js:6]: Again, maybe delete?

File: src/style.scss [hunk 4]
```diff
@@ -34,7 +35,7 @@ body {
 }
 
 .pcui-menu {
-    font-size: 13px;
+    // font-size: 13px;
```
Comment (willeastcott) [inline src/style.scss:38]: Delete

**Review by willeastcott — APPROVED** [review]
Approving with some super-minor feedback. You might have to bump Rollup back down to 2.x until related plugins are updated.

File: package.json (hunk 1 above)
Comment (slimbuck) [reply package.json:31]: thanks, removed

File: rollup.config.mjs (hunk 2 above)
Comment (slimbuck) [reply rollup.config.mjs:37]: done, thanks

File: src/index.js (hunk 3 above)
Comment (slimbuck) [reply src/index.js:6]: These were removed temporarily till I have time to get these package versions.

File: src/style.scss (hunk 4 above)
Comment (slimbuck) [reply src/style.scss:38]: done, thanks

**Pull request merged**